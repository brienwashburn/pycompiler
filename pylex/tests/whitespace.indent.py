def True:
  class
  ()
   1234
