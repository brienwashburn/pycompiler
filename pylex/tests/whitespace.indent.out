(KEYWORD def)
(KEYWORD True)
(PUNCT ":")
(NEWLINE)
(INDENT)
(KEYWORD class)
(NEWLINE)
(PUNCT "(")
(PUNCT ")")
(NEWLINE)
(INDENT)
(LIT 1234)
(NEWLINE)
(DEDENT)
(DEDENT)
(ENDMARKER)
